src/styles/index.scss

```diff
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import './blocks/utils/varible';
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), file names should be kebab-case. The file name 'varible' contains a typo and should be 'variables' instead.